src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -484,11 +505,17 @@ private static bool CheckPipeConnectionOwnership(NamedPipeClientStream pipeStrea
             }
         }
 
-        private static ObjectSecurity GetPipeSecurity(PipeStream pipeStream) =>
-            (ObjectSecurity)typeof(PipeStream)
-            .GetTypeInfo()
-            .GetDeclaredMethod("GetAccessControl")
-            ?.Invoke(pipeStream, parameters: null);
+        private static ObjectSecurity GetPipeSecurity(PipeStream pipeStream)
+        {
+#if NETSTANDARD1_3
```

Comment:
If the build task is multi-targeted now, why do we have a netstandard1.3 version?